inventory/prod/group_vars/all.yaml

```diff
@@ -7,3 +7,4 @@ os_cloud_profile: devstack
 terraform_base_dir: terraform
 key_name: key_csm_controller
 requirements: requirements.txt
+cloud_config: /etc/openstack
```

Comment:
You can use `os_cloud_config_file` with  `| dirname` filter, as those two values are tightly bidden